README.md

```diff
@@ -238,13 +236,6 @@ The BSD 3-Clause License
 We have a gitter rooms for discussing new features & QA.
 Feel free to join us!
 
-<table>
-<tr>
-    <td><b> developers </b></td>
-    <td> https://gitter.im/tiny-dnn/developers </td>
-</tr>
-<tr>
-    <td><b> users </b></td>
-    <td> https://gitter.im/tiny-dnn/users </td>
-</tr>
-</table>
```

Comment:
The previous was done in purpose to improve readability. Not consistent shape to me